### ISSUE TITLE
Same handling for bikes & wheelchairs in vehicles_ft?

### ISSUE THREAD

**Issue by ddorinson** [osplanning-data-standards/GTFS-PLUS#54]
Hi @danielsclint , @lmz ,@e-lo -- I am making a final pass to confirm that the T2 memo is consistent with the GTFS-PLUS repo, and I see a possible internal inconsistency **within** [vehicles_ft.txt](https://github.com/osplanning-data-standards/GTFS-PLUS/blob/master/files/vehicles_ft.md).  Looking at the descriptions for `wheelchair_capacity` and `bicycle_capacity` in that file, it seems like they are set-up for parallel treatment, but they don't quite match:
1. Only `wheelchair_capacity` explicitly says it will override the value "in trip file" (presumably [trips.txt](https://github.com/osplanning-data-standards/GTFS-PLUS/blob/master/files/trips.md)?).  Is that also true for `bicycle_capacity`?  (If not, should it be?)
2. Only `bicycle_capacity` explains that a blank in this field should _not_ be treated as infinity if the trip file specifies no bikes.  Should `wheelchair_capacity` also be zero (not infinity) if trips.txt says wheelchairs cannot be accommodated?

Perhaps these differences are handled in the way the files are read in, but that doesn't seem like our usual M.O., and I couldn't find anything in the Fast-Trips code to confirm that the fields are actually treated any differently in the calcs.  I would like to submit a pull request here in the GTFS-PLUS repo to clarify, but it should reflect how the code actually works, so wanted to double-check with you.  Thoughts?

**Comment by e-lo** [comment]
@ddorinson  – Thanks for checking on this.
tl;dr I don't think we need to make any changes.

- Not all of the definitions in GTFS-PLUS are handled explicitly in Fast-Trips, but I think we decided that we should still be explicit about what overrides what....so I think we are OK if Fast-Trips doesn't have a handler for these (for now).
 - Per ADA and general practice, wheelchairs are generally accommodated while bikes are generally not, so I think it is OK to have their blanks default to different values.  In fact many operators will make others disembark in order to accommodate another wheelchair, so I the infinity default makes sense.
 - The reason why the `wheelchair_capacity` in `vehicles_ft.txt` is set up to override `trips.txt` is that `wheelchair_capacity` is contained in the official GTFS data standard, but it is more efficient to specify it at the vehicle- rather than trip-level.  There is no field for bicycle capacity in the official GTFS `trips.txt` standard so specifying an override is moot.

**Comment by ddorinson** [comment]
@e-lo -- Thanks for these details, which help fill in some of the gaps for me.  For the record, I do not see wheelchair-**capacity** in the GTFS standard for `trips.txt`, only a wheelchair-**accessible** indicator variable with three options: 0=unknown; 1=1+ WC allowed; 2=no WC allowed.  So the idea is that a blank in the `wheelchair_capacity` field in `vehicles_ft.txt` will override `wheelchair-accessible`=2 in `trips.txt`.  In contrast, a blank field for `bicycle_capacity` in `vehicles_ft.txt` cannot override `bicycles-allowed`=2 in `trips.txt`.  I will make sure the table in Appendix B reflects this approach.  Thx again!
